plugins/linear-genome-view/src/BaseLinearDisplay/models/serverSideRenderedBlock.ts

```diff
@@ -240,6 +242,7 @@ export function renderBlockData(
         adapterConfig,
         rendererType: rendererType.name,
         sessionId,
+        layoutId,
```

Comment:
we add a new RPC parameter `layoutId` that helps with async feature details when more than 1 view is displaying the same track (e.g. breakpoint split view). this helps because if more than one view was displaying the same track (as in breakpoint split view) the cachedLayout was getting tossed out for one or the other. now, multiple layouts can be stored on the worker